content/docs/csidriver/features/powerflex.md

```diff
@@ -907,6 +907,20 @@ allowedTopologies:
       values:
         - "true"
 ```
+## Configuring custom access to NFS exports
```

Comment:
Can you please add 'externalAccess' here: https://dell.github.io/csm-docs/docs/csidriver/installation/helm/powerflex/#install-the-driver.  Also, `X_CSI_POWERFLEX_EXTERNAL_ACCESS` to https://dell.github.io/csm-docs/docs/csidriver/installation/operator/powerflex/#install-driver.

Reply:
Done